### PR TITLE
build: remove vue-loader

### PR DESCRIPTION
**Pull request by JadenSimon**
It's not doing anything currently and it has a security vulnerability, albeit a trivial one.
~TODO: test webviews on C9~
Tested, debug config editor shows up correctly.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by JadenSimon** [comment]
Windows insider build keeps failing on a test completely unrelated to webviews.

File: package.json [hunk 1]
```diff
@@ -1510,8 +1510,7 @@
         "copyNonCodeFiles": "node ./build-scripts/copyNonCodeFiles.js",
         "copyExtensionMain": "node ./build-scripts/copyExtensionMain.js",
         "copyDistFiles": "ts-node ./build-scripts/copyDistFiles.ts",
-        "buildScripts": "npm run bundleDeps && npm run copyExtensionMain && npm run copyDistFiles && npm run copyNonCodeFiles && npm run generateNonCodeFiles && npm run buildVue",
-        "buildVue": "webpack --config-name vueLoader",
```
Comment (bryceitoc9) [inline package.json:1514]: You'll still want a separate build for webviews. Remember: webviews need to be built separately for frontend use (backend import statements don't work on the frontend). Unsure if vue-loader is necessary.

File: webpack.config.js [hunk 2]
```diff
@@ -94,45 +95,4 @@ const baseConfig = {
...
-                options: {
-                    loader: 'tsx',
-                    target: 'es2018',
-                },
-                exclude: /node_modules/,
-            },
-            {
-                test: /\.vue$/,
-                loader: 'vue-loader',
-            },
```
Comment (bryceitoc9) [inline webpack.config.js:132]: I think the only valuable part of this PR is right here. The rest of the job is using `esbuild-loader` and will be necessary.

If you're tripping on the name, just rename it to something like `compileWebviews`

**Review by bryceitoc9 — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 2 above)
Comment (JadenSimon) [reply webpack.config.js:132]: Maybe I'm missing something but the webviews are being compiled separately in this PR. Compilation is based off the entry-points, so the webviews would still be separate from everything else.

Comment (bryceitoc9) [reply webpack.config.js:132]: Maybe it was fixed with the move from esbuild to webpack? I know that that had issues with the target type.

Comment (JadenSimon) [reply webpack.config.js:132]: Well let's just wait for a bit on this PR to figure out what exactly is happening with these webviews. Insiders test is broken anyway.

Comment (bryceitoc9) [reply webpack.config.js:132]: Confirmed in a separate branch that this works following an `npm run compile` (I guess esbuild handles frontend + backend better than webpack did).

**Review by bryceitoc9 — APPROVED** [review]
May still be worth investigating an alternate faster way to build webviews (since they'll only update with an npm run compile). That being said, that's now faster than webpacking was previously.

**Review by justinmk3 — APPROVED** [review]
🚀 🎉

**Pull request merged**